File: wrappers/mvc/demos/Kendo.Mvc.Examples/Controllers/Scatter_Charts/Local_Data_BindingController.cs
using System;
using System.Linq;
using System.Web.Mvc;
using Kendo.Mvc.Examples.Models;

namespace Kendo.Mvc.Examples.Controllers
{
    public partial class Scatter_ChartsController : Controller
    {
        public ActionResult Local_Data()
        {
            return View(ChartDataRepository.PricePerformanceData());
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/scattercharts/LocalDataBindingController.java
package com.kendoui.spring.controllers.scattercharts;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import com.kendoui.spring.models.ChartDataRepository;

@Controller("dataviz-scatter_chart-local_data-controller")
@RequestMapping(value="/scatter-charts/")
public class LocalDataBindingController {
    @RequestMapping(value = "/local-data-binding", method = RequestMethod.GET)
    public String index(Model model) {
        model.addAttribute("pricePerfomance", ChartDataRepository.PricePerformanceData());
     
        return "/scatter-charts/local-data-binding";
    }
}
